src/phasorpy/io.py

```diff
@@ -522,6 +540,266 @@ def read_sdt(
     return DataArray(data, **metadata)
 
 
+def read_ptu(
+    filename: str | PathLike[Any],
+    /,
+    selection: Sequence[int | slice | EllipsisType | None] | None = None,
```

Comment:
I didn't test the selection argument because I had some trouble understanding what it is and the different options.

The other arguments were tested and worked as expected

Reply:
The `selection` parameter allows numpy style indexing with slices, integers, and ellipsis, except that step sizes mean integration. See for example https://github.com/cgohlke/ptufile/blob/3a2741fd3b21101d91f55ace8c55de0f5d781135/ptufile/ptufile.py#L172-L178 and https://github.com/cgohlke/ptufile/blob/3a2741fd3b21101d91f55ace8c55de0f5d781135/tests/test_ptufile.py#L451-L461. It's an advanced option that would allow, for example, to define the maximum number of bins returned when used with `trimdims`.